packages/zoe/src/instanceRecordStorage.js

```diff
@@ -28,8 +28,6 @@ const { ownKeys } = Reflect;
  * @returns {(ir: InstanceRecord) => InstanceState}
  */
 export const makeInstanceRecordStorage = baggage => {
-  provide(baggage, 'instanceRecord', () => undefined);
```

Comment:
Any idea why this was there? Likewise with the other pure deletions of unneeded baggage stuff?

Reply:
Just leftover. Explained in the commit: [19407b8](https://github.com/Agoric/agoric-sdk/pull/9661/commits/19407b8930ee496aaf6f8994820c6a2ec6931ad0)